### ISSUE TITLE
Certificate Authority

### ISSUE THREAD

**Issue by datamuc** [Griesbacher/nagflux#40]
https://github.com/Griesbacher/nagflux/blob/ea877539bc49ed67e9a5e35b8a127b1ff4cadaad/config/Config.go#L38

If the influxdb connection is secured by TLS, i guess we need an option to provide CA certificates. For example to provide a company's own CA. This may also be true for ElasticSearch.

**Comment by Griesbacher** [comment]
Yes, that might help in some cases. Currently all certificates are accepted: https://github.com/Griesbacher/nagflux/blob/master/target/influx/Connector.go#L46
